diff --git a/node_modules/@react-navigation/drawer/src/views/DrawerView.tsx b/node_modules/@react-navigation/drawer/src/views/DrawerView.tsx
index 94770c4..bab6d32 100644
--- a/node_modules/@react-navigation/drawer/src/views/DrawerView.tsx
+++ b/node_modules/@react-navigation/drawer/src/views/DrawerView.tsx
@@ -75,24 +75,10 @@ function DrawerViewBase({
   detachInactiveScreens = Platform.OS === 'web' ||
     Platform.OS === 'android' ||
     Platform.OS === 'ios',
-  // Reanimated 2 is not configured
-  // @ts-expect-error: the type definitions are incomplete
-  useLegacyImplementation = !Reanimated.isConfigured?.(),
-}: Props) {
-  // Reanimated v3 dropped legacy v1 syntax
-  const legacyImplemenationNotAvailable =
-    require('react-native-reanimated').abs === undefined;
   
-  if (useLegacyImplementation && legacyImplemenationNotAvailable) {
-    throw new Error(
-      'The `useLegacyImplementation` prop is not available with Reanimated 3 as it no longer includes support for Reanimated 1 legacy API. Remove the `useLegacyImplementation` prop from `Drawer.Navigator` to be able to use it.'
-    );
-  }
-
-  const Drawer: React.ComponentType<DrawerProps> = useLegacyImplementation
-    ? require('./legacy/Drawer').default
-    : require('./modern/Drawer').default;
+}: Props) {
   
+const Drawer = require('./modern/Drawer').default;
   const focusedRouteKey = state.routes[state.index].key;
   const {
     drawerHideStatusBarOnOpen = false,
@@ -225,6 +211,7 @@ function DrawerViewBase({
 
           return (
             <MaybeScreen
+            // @ts-ignore
               key={route.key}
               style={[StyleSheet.absoluteFill, { zIndex: isFocused ? 0 : -1 }]}
               visible={isFocused}
